
From: Paul Mackerras <paulus@samba.org>

David Engebretsen has moved on to other things and is no longer maintaining
ppc64.  This patch adds an entry in CREDITS to note his contribution in
leading the team that did the PPC64 port originally and updates various
PPC-related MAINTAINERS entries.

Signed-off-by: Paul Mackerras <paulus@samba.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/CREDITS     |    4 ++++
 25-akpm/MAINTAINERS |   13 ++++++++-----
 2 files changed, 12 insertions(+), 5 deletions(-)

diff -puN CREDITS~update-ppc-maintainers-credits CREDITS
--- 25/CREDITS~update-ppc-maintainers-credits	2004-08-30 00:50:16.730221024 -0700
+++ 25-akpm/CREDITS	2004-08-30 00:50:16.737219960 -0700
@@ -911,6 +911,10 @@ S: Brevia 1043
 S: S-114 79 Stockholm
 S: Sweden
 
+N: David Engebretsen
+E: engebret@us.ibm.com
+D: Linux port to 64-bit PowerPC architecture
+
 N: Michael Engel
 E: engel@unix-ag.org
 D: DECstation framebuffer drivers
diff -puN MAINTAINERS~update-ppc-maintainers-credits MAINTAINERS
--- 25/MAINTAINERS~update-ppc-maintainers-credits	2004-08-30 00:50:16.732220720 -0700
+++ 25-akpm/MAINTAINERS	2004-08-30 00:50:16.739219656 -0700
@@ -1296,13 +1296,14 @@ S:	Maintained
 LINUX FOR POWERPC
 P:	Paul Mackerras
 M:	paulus@samba.org
-W:	http://www.fsmlabs.com/linuxppcbk.html
+W:	http://www.penguinppc.org/
+L:	linuxppc-dev@lists.linuxppc.org
 S:	Supported
 
 LINUX FOR POWER MACINTOSH
 P:	Benjamin Herrenschmidt
 M:	benh@kernel.crashing.org
-W:	http://www.linuxppc.org/
+W:	http://www.penguinppc.org/
 L:	linuxppc-dev@lists.linuxppc.org
 S:	Maintained
 
@@ -1333,9 +1334,11 @@ M:	acme@conectiva.com.br
 S:	Maintained
 
 LINUX FOR 64BIT POWERPC
-P:	David Engebretsen (stable kernel)
-M:	engebret@us.ibm.com
-P:	Anton Blanchard (development kernel)
+P:	Paul Mackerras
+M:	paulus@samba.org
+M:	paulus@au.ibm.com
+P:	Anton Blanchard
+M:	anton@samba.org
 M:	anton@au.ibm.com
 W:	http://linuxppc64.org
 L:	linuxppc64-dev@lists.linuxppc.org
_
